create_infotexts.py

```diff
@@ -384,8 +385,6 @@ def process_depicted_place(self, place_string, places_mapping):
         if "commons" in places_mapping[place_string].keys() and places_mapping[place_string]["commons"] != "-":
             self.content_cats.append(places_mapping[place_string]["commons"])
 
-        self.data["depicted_place"] = place_as_wikitext
```

Comment:
You cannot remove this. You need this after all of the if/else to actually set the value. What you should do instead is to ensure you just use `place_as_wikitext` not `self.data["depicted_place"]` above (with the exception of the one with the `return`